networking-calico/setup.cfg

```diff
@@ -1,11 +1,11 @@
 [metadata]
 name = networking-calico
 summary = This project contains documentation and code for integrating Project Calico networking into OpenStack/Neutron.
-description-file =
+description_file =
     README.md
 author = Project Calico
-author-email = neil@projectcalico.org
-home-page = https://www.projectcalico.org/
+author_email = neil@projectcalico.org
```

Comment:
I'm not super familiar with this - @neiljerram could you take a look at this particular diff and make sure it makes sense?